src/main/java/org/ideoholic/curium/model/parents/dto/Parents.java

```diff
@@ -2,288 +2,106 @@
-	}
-
-	public void setCocontactnumber(String cocontactnumber) {
-		this.cocontactnumber = cocontactnumber;
-	}
+	private String contactnumber;
 
 	@Column(name = "contactnumber")
-	public String getContactnumber() {
-		return contactnumber;
-	}
-
-	public void setContactnumber(String contactnumber) {
-		this.contactnumber = contactnumber;
-	}
+	private String cocontactnumber;
 
-	/**
-	 * @return the email
-	 */
 	@Column(name = "email")
-	public String getEmail() {
-		return email;
-	}
-
-	/**
-	 * @param email the email to set
-	 */
-	public void setEmail(String email) {
-		this.email = email;
-	}
-
-	public Student getStudent() {
-		return student;
-	}
+	private String email;
 
-	public void setStudent(Student student) {
-		this.student = student;
-	}
+	@ManyToOne(cascade = CascadeType.ALL, fetch=FetchType.LAZY)
+	@JoinColumn(name = "sid", unique=true, nullable = false)
+	private Student student;
 
 	@Column(name = "branchid")
-	public int getBranchid() {
-		return branchid;
-	}
-
-	public void setBranchid(int branchid) {
-		this.branchid = branchid;
-	}
+	private int branchid;
 
 	@Column(name = "fathersqualification")
-	public String getFathersqualification() {
-		return this.fathersqualification;
-	}
-
-	public void setFathersqualification(String fathersqualification) {
-		this.fathersqualification = fathersqualification;
-	}
+	private String fathersqualification;
 
 	@Column(name = "mothersqualification")
-	public String getMothersqualification() {
-		return this.mothersqualification;
-	}
-
-	public void setMothersqualification(String mothersqualification) {
-		this.mothersqualification = mothersqualification;
-	}
-
-	public String getFatherscastecertno() {
-		return fatherscastecertno;
-	}
-
-	public void setFatherscastecertno(String fatherscastecertno) {
-		this.fatherscastecertno = fatherscastecertno;
-	}
-
-	public String getMotherscastecertno() {
-		return motherscastecertno;
-	}
-
-	public void setMotherscastecertno(String motherscastecertno) {
-		this.motherscastecertno = motherscastecertno;
-	}
+	private String mothersqualification;
 
-	public String getFatherscaste() {
-		return fatherscaste;
-	}
+	@Column(name = "motherscastecertno", length = 25)
+	private String motherscastecertno;
 
-	public void setFatherscaste(String fatherscaste) {
-		this.fatherscaste = fatherscaste;
-	}
+	@Column(name = "fatherscastecertno", length = 25)
+	private String fatherscastecertno;
 
-	public String getMotherscaste() {
-		return motherscaste;
-	}
+	@Column(name = "fatherscaste", length = 25)
+	private String fatherscaste;
 
-	public void setMotherscaste(String motherscaste) {
-		this.motherscaste = motherscaste;
-	}
+	@Column(name = "motherscaste", length = 25)
+	private String motherscaste;
 
-	public int getUserid() {
-		return userid;
-	}
+	@Column(name = "userid")
+	private int userid;
 
-	public void setUserid(int userid) {
-		this.userid = userid;
-	}
+	private String emergencycontactno;
+	private String fatheroccupation;
```

Comment:
```suggestion
        private String emergencycontactno;
	private String fatheroccupation;
```
Noteing that these two fields are not mapped